### ISSUE TITLE
Please consider moving away from SourceForge

### ISSUE THREAD

**Issue by noloader** [libexpat/libexpat#157]
I'm trying to build GnuTLS from source code. GnuTLS  depends on libunbound , and libunbound has a dependency on libexpat.

I found libexpat's Git.io site at https://libexpat.github.io/. When I follow the download link, it takes me to SourceForge, which was specifically avoided from the search results. SourceForge is known to tamper with packages. Its been well reported around the web.

SourceForge  is an untrusted source and it creates a governance problem for us. We want to build the software using authentic sources with established provenance, but we can't because we know what SourceForge does.

Please consider moving away from SourceForge.

**Comment by hartwork** [comment]
The downloads have not been moved to GitHub _yet_ for practical reasons, priorities, time constraints. Related tickets are #1 and #156. It was planned from the beginning to move downloads to GitHub at some point — it still is.

**Comment by noloader** [comment]
Thanks everyone. For completeness, I help out with a few free and open software projects. We eat the same dogfood :)

Here's the old [Crypto++ SourceForge](https://sourceforge.net/projects/cryptopp/) site. We retained the site for historical purposes. But we changed the project's description to explain we moved from SourceForge to GitHub due to developer demands. We also added a `README.md` to to the download area with the same (copy/paste) message.

It looks like this is being tracked already. I'll close the request because I like tidy bug trackers.

Thanks, and sorry again for speaking out.  I meant no disrespect.
